client-java/pom.xml

```diff
@@ -147,6 +144,7 @@
 
         <!-- TEST -->
 
+        <!-- Micronaut Dependencies -->
         <dependency>
             <groupId>io.micronaut.test</groupId>
             <artifactId>micronaut-test-junit5</artifactId>
```

Comment:
Reads like there is an exclusion of part of JUnit5 from the JUnit5 testing.  Is this necessary?

Reply:
We don't want the version of JUnit5 in the micronaut dependency to override the one we're using.